rwkv.cpp

```diff
@@ -413,26 +413,38 @@ void rwkv_max_impl(const int n_cols, float * dest, const float * src0, const flo
     }
 }
 
-struct ggml_tensor * rwkv_exp(ggml_context * ctx, struct ggml_tensor * x) {
+struct ggml_tensor * rwkv_op_inplace(struct ggml_context * ctx, enum ggml_op op, struct ggml_tensor * a, struct ggml_tensor * b) {
```

Comment:
>I don't think the memory savings are that worth it

I agree and would prefer to code to be simpler, at the cost of being slightly less optimal. The bottleneck is in matmul anyway, no point to optimize activation functions